robot/EDA/resources/locators.py

```diff
@@ -144,7 +144,7 @@
     "eda_tile": "//div[@class='slds-app-launcher__tile-body']//a[contains(text(),'EDA')]/../..",
     "new_account": "//span[@title='New Account']",
     "new_account_next_button": "//button[contains(@class, 'slds-button')]//span[@class=' label bBody' and text()='Next']",
-    "new_account_name": "//input[@class='input uiInput uiInputText uiInput--default uiInput--input']",
+    "new_account_name": "//label/span[text()='Account Name']/following-sibling::span/following::input[1]",
```

Comment:
@dashattuck Same comment that I made earlier in another open PR - if this is being handled in #968, then let's remove it from here. I'd prefer to keep Robot changes separate from other work items.

Reply:
Reverted, resolving.